.github/workflows/build-windows-x86_64.yml

```diff
@@ -86,14 +90,53 @@ jobs:
       # Use a retry action because inexplicably sometimes stack install stalls
       # indefinitely after the `compiling Main` step, but then succeeds immediately
       # with a second stack install. 🤷🏻‍♂️
-      - uses: nick-invision/retry@v2
+      - uses: nick-fields/retry@v2
         with:
           timeout_minutes: 5
-          max_attempts: 3
-          command: stack install
+          max_attempts: 4
+          # https://gitlab.haskell.org/ghc/ghc/-/issues/20878
+          # https://gitlab.haskell.org/ghc/ghc/-/issues/20010#note_359766
+          command: stack install --ghc-options '-optl"-Wl,-Bstatic,-lstdc++,-lgcc_s,-lwinpthread,-Bdynamic"'
 
-      - name: Archive elm-dev.exe artifact
-        uses: actions/upload-artifact@v2
+      - name: Copy binary files, dlls & check
+        run: |
+          mkdir elm-dev
+          cp ~/AppData/Roaming/local/bin/elm-dev.exe elm-dev/elm-dev.exe
+          cp distribution/dlls/* elm-dev/
+          ./elm-dev/elm-dev.exe --version-full
```

Comment:
Not 100% sure if we'll need this step for elm-dev, it could be that Lamdera uses more libs due to its features. So I've kept the DLL's out for now until we can confirm if the produced Windows binary has issues.

Reply:
Makes sense to me!